src/main/java/mate/academy/spring/mapper/CinemaHallMapper.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.mapper;
+
+import mate.academy.spring.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.CinemaHallResponseDto;
+import mate.academy.spring.model.CinemaHall;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallMapper {
+    public CinemaHallResponseDto parse(CinemaHall cinemaHall) {
```

Comment:
We have to use method name toDto, it is in our checklist
The same in other mappers
```suggestion
    public CinemaHallResponseDto toDto(CinemaHall cinemaHall) {
```